plugins/woocommerce/src/Internal/Traits/OrderAttributionMeta.php

```diff
@@ -385,13 +385,24 @@ private function get_field_description( string $field ): string {
 	 * @return array Order count, total spend, and average spend per order.
 	 */
 	private function get_customer_history( $customer_identifier ): array {
+		/*
+		 * Exclude the statuses that aren't valid for the Customers report.
+		 * 'checkout-draft' is the checkout block's draft order status.
+		 * @see /Automattic/WooCommerce/Admin/API/Report/DataStore::get_excluded_report_order_statuses()
+		 */
+		$all_statuses_no_prefix = array_map(
+			function ( $status ) {
+				return str_replace( 'wc-', '', $status );
+			},
+			array_keys( wc_get_order_statuses() )
+		);
```

Comment:
Or ( ofc `use` should be elsewhere ) :) 

```suggestion
		use Automattic\WooCommerce\Admin\API\Reports\Controller;
		$all_statuses_no_prefix = Controller::get_order_status_labels()
```

Reply:
That's good, although since we don't need the labels, I went with `ReportsController::get_order_statuses()` (3d39899b86f463c044821903b192cdedfd5754fb)